resources/js/ResizeSensor.js

```
@@ -8,210 +8,210 @@
+  ResizeSensor.detach = function(element, ev) {
+    forEachElement(element, function(elem){
+      if (!elem) return
+      if(elem.resizedAttached && typeof ev == "function"){
+        elem.resizedAttached.remove(ev);
+        if(elem.resizedAttached.length()) return;
+      }
+      if (elem.resizeSensor) {
+        if (elem.contains(elem.resizeSensor)) {
+          elem.removeChild(elem.resizeSensor);
+        }
+        delete elem.resizeSensor;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)